test/plugins/link.js

```diff
@@ -26,7 +29,7 @@ describe('Link plugin', function() {
     });
 
     this.irc.createMessage({
-      message: 'http://localhost:9002/basic'
+      message: 'http://127.0.0.1:9002/basic'
```

Comment:
@williamboman, could you amend and remove that from second commit so that we keep `localhost`? [This commit](https://github.com/erming/shout/commit/58aa236dcd6df1011a239568201da41b893e9f84) now makes this unnecessary.